usaspending_api/references/tests/test_glossary_autocomplete.py

```diff
@@ -42,6 +42,45 @@ def test_glossary_autocomplete(client, glossary_data, fields, value, expected):
     check_autocomplete('references/glossary', client, fields, value, expected)
 
 
+@pytest.mark.django_db
+def test_glossary_v2_autocomplete(client):
+    mommy.make(
+        Definition,
+        term='Abacus',
+        slug="ab",)
+    mommy.make(
+        Definition,
+        term="Aardvark",
+        slug="aa")
+
+    resp = client.post(
+        '/api/v2/autocomplete/glossary/',
+        content_type='application/json',
+        data=json.dumps({"search_text": "ab"}))
+    json_response = json.loads(resp.content.decode("utf-8"))
+    assert resp.status_code == status.HTTP_200_OK
+    assert json_response['results']['search_text'] == "ab"
+    assert json_response['results']['term'] == ["Abacus"]
+
+    resp = client.post(
+        '/api/v2/autocomplete/glossary/',
+        content_type='application/json',
+        data=json.dumps({"search_text": "aa"}))
+    json_response = json.loads(resp.content.decode("utf-8"))
+    assert resp.status_code == status.HTTP_200_OK
+    assert json_response['results']['search_text'] == "aa"
+    assert json_response['results']['term'] == ["Aardvark"]
+
+    resp = client.post(
+        '/api/v2/autocomplete/glossary/',
+        content_type='application/json',
+        data=json.dumps({"search_text": "a"}))
+    json_response = json.loads(resp.content.decode("utf-8"))
+    assert resp.status_code == status.HTTP_200_OK
+    assert json_response['results']['search_text'] == "a"
+    assert sorted(json_response['results']['term']) == ["Aardvark", "Abacus"]
+
```

Comment:
One could also test the case of typing in "b" to see if nothing is returned, along with a string like "abb" that goes past the auto correction.